plugins/kimspect.yaml

```
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kimspect
+spec:
+  version: v0.0.35
+  shortDescription: inspect container images on your pods and nodes.
```

in a subsequent PR please consider doing
```suggestion
  shortDescription: Inspect container images on your pods and nodes
```